### PR TITLE
action: refactor serverless params

### PR DESCRIPTION
**Pull request by v1v**
## What does this PR do?

Use script for preparing the oblt-cli variables

## Why is it important?

Platform agnostic

## Related issues
Closes #ISSUE

## Test

image

### CODE REVIEW

File: .github/actions/oblt-cli-create-serverless/action.yml [hunk 1]
```diff
@@ -37,71 +37,50 @@ inputs:
...
+                parameters.GitHubIssue = context.issue.number
+              }
+            } else {
+              parameters.GitHubPullRequest = context.issue.number
+            }
+          }
+          console.log(`Create PARAMETERS env variable`)
+          core.exportVariable('PARAMETERS', `--parameters='${(JSON.stringify(parameters))}'`)
+      env:
+        cluster_name_prefix: ${{ inputs.cluster-name-prefix }}
```
Comment (v1v) [inline .github/actions/oblt-cli-create-serverless/action.yml:74]: I could not use `process.env` since those variables contain some `hyphens` and somehow I faced some errors. So I decided to use the env approach to solve my initial problem and also to protect from any input injections, as explained in https://docs.github.com/en/actions/security-guides/security-hardening-for-github-actions#using-an-intermediate-environment-variable

File: .github/actions/oblt-cli-create-serverless/action.yml [hunk 2]
```diff
@@ -37,71 +37,50 @@ inputs:
...
-      shell: bash
-
-    - name: Configure oblt-cli flags for gitops - pull-request
-      if: github.event.issue.pull_request && contains(inputs.gitops, 'true')
-      run: |
-        {
-          echo -n '{'
-          echo -n '"GitOps":"true",'
-          echo -n '"GitHubRepository":"${{ github.repository }}",'
-          echo -n '"GitHubCommit":"${{ github.event.comment.id }}",'
```
Comment (v1v) [inline .github/actions/oblt-cli-create-serverless/action.yml:76]: This is not supported, I could not find where we use it so far. And I could not find how to gather the sha commit. Somehow I feel we don't use it and we don't need it anyway

**Review by reakaleek — APPROVED** [review]
LGTM,

in general, I was wondering, if we could put those github-scripts into a JS file, so we can benefit from syntax highlighting and code completion, thus increasing the maintainability?

**Comment by v1v** [comment]
> if we could put those github-scripts into a JS file
I could not find how to use `${{ github.action_path }}` in https://github.com/actions/github-script . But it seems I can use `GITHUB_ACTION_PATH`

Let me try again

**Comment by reakaleek** [comment]
> > if we could put those github-scripts into a JS file
> 
> I could not find how to use `${{ github.action_path }}` in https://github.com/actions/github-script . But it seems I can use `GITHUB_ACTION_PATH`
> 
> Let me try again

I found this: https://github.com/actions/github-script#run-a-separate-file

**Comment by reakaleek** [comment]
> > > if we could put those github-scripts into a JS file
> > 
> > 
> > I could not find how to use `${{ github.action_path }}` in https://github.com/actions/github-script . But it seems I can use `GITHUB_ACTION_PATH`
> > Let me try again
> 
> I found this: https://github.com/actions/github-script#run-a-separate-file

Actually, this seems to make it more complicated.

> Additionally, you'll want to use the [checkout action](https://github.com/actions/checkout) to make sure your script file is available.
